pkg/downloader/utils.go

```diff
@@ -59,40 +59,41 @@ func FindPackageByModSpec(root string, modSpec *ModSpec) (string, error) {
 		if err != nil {
 			return err
 		}
-		if info.IsDir() {
-			kclModPath := filepath.Join(path, constants.KCL_MOD)
-			if _, err := os.Stat(kclModPath); err == nil {
-				// If the package name and version are specified,
-				// we can directly check if the kcl.mod file matches the package.
-				if matchesPackageSpec(kclModPath, modSpec) {
-					result = path
-					return filepath.SkipAll
-				} else if modSpec.Version == "" {
-					// If the package name specified, but version are not specified,
-					if utils.MatchesPackageName(kclModPath, modSpec.Name) {
-						// load the version from the kcl.mod file
-						tmpSpec, err := loadModSpecFromKclMod(kclModPath)
-						if err != nil {
-							return err
-						}
-						// Remember the local path with the highest version
-						tmpVer, err := version.NewSemver(tmpSpec.Version)
+		if !info.IsDir() {
+			return nil
+		}
```

Comment:
It's just a simple code cleaning, returning the if condition in advance without any logical changes

Reply:
seems like this is the reason of CI failed, reverted it.